main.py

```diff
@@ -21,6 +21,22 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 
+import os
+from os.path import join
+import sys
+
+# import dataclasses and typchecking
+from dataclasses import dataclass, field
+from typing import List
+
+# import pygame related
+import pygame
+from pytmx.util_pygame import load_pygame
+
+# import Python specific objects, functions and functionality
+# from src.py_version.board import Board
+# from src.py_version.player import Player
+from button import Button
 
 # import Pygame specific objects, functions and functionality
 from src.GUI.gameloop import GUI
```

Comment:
This import is where the GUI class is stored for context, it is called gameloop.py